### PR TITLE
ImageryView 🖼️ + IndependentTimeConductor ⏰

### LINKED ISSUE #6434: Image view should support showing more images than the conductor bounds
**Is your feature request related to a problem? Please describe.**
Users would prefer if Imagery in the thumb strip persisted and was not removed by the Time Conductor bounds, while using a designated "now" context (local clock, remote clock). This supports the usage scenario of wanting to globally see plotted data for around 15 minutes, but imagery for much longer.

**Describe the solution you'd like**
1. A toggle switch for Imagery views, per view and enabled by default, that makes that view ignore the TC real-time bounds. "Now" would still be defined by the TC.
Discuss if Fixed Timespan mode would or would not pay attention to this setting.
1. Independent Time Conductor is another possibility, but that UX is more cumbersome to deal with for this use case; option 1 above would be preferred.

**Additional context**
For images it's good enough to just enable the independent time conductor for now.

### PR DESCRIPTION
**Pull request by scottbell**
**Waiting on [Time Conductor PR](https://github.com/nasa/openmct/pull/6768)**

Closes #6434

### Describe your changes:
Adds the Independent Time Conductor to the ImageryView. Fixes TelemetryCollection to respond properly to time conductor changes.

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [x] Tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Reviewer has tested changes by following the provided instructions?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate automated tests included?
* [ ] Code style and in-line documentation are appropriate?
* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

File: src/plugins/imagery/components/ImageryView.vue [hunk 1]
```diff
@@ -377,9 +377,9 @@ export default {
     canTrackDuration() {
       let hasClock;
       if (this.timeContext) {
-        hasClock = this.timeContext.clock();
+        hasClock = this.timeContext.getClock();
```
Comment (jvigliotta) [inline src/plugins/imagery/components/ImageryView.vue:380]: I think we'll want to use the new `this.openmct.time.isRealTime()` method here.

File: src/plugins/imagery/components/ImageryView.vue [hunk 2]
```diff
@@ -533,9 +533,9 @@ export default {
     isFixed() {
       let clock;
       if (this.timeContext) {
-        clock = this.timeContext.clock();
+        clock = this.timeContext.getClock();
       } else {
-        clock = this.openmct.time.clock();
+        clock = this.openmct.time.getClock();
```
Comment (jvigliotta) [inline src/plugins/imagery/components/ImageryView.vue:538]: You can just return the new `this.openmct.time.isFixed()` method here.

File: src/plugins/imagery/components/ImageryView.vue [hunk 3]
```diff
@@ -1111,7 +1111,7 @@ export default {
       window.clearInterval(this.durationTracker);
     },
     updateDuration() {
-      let currentTime = this.timeContext.clock() && this.timeContext.clock().currentValue();
+      let currentTime = this.timeContext.getClock() && this.timeContext.getClock().currentValue();
```
Comment (jvigliotta) [inline src/plugins/imagery/components/ImageryView.vue:1114]: You should no longer need to check for a clock (I think this isn't called if it isn't in realtime anyways), there is always a clock (even in fixed mode now, which should be irrelevant for this logic).

File: src/ui/layout/BrowseBar.vue [hunk 4]
```diff
@@ -142,7 +142,13 @@ import NotebookMenuSwitcher from '@/plugins/notebook/components/NotebookMenuSwit
 import IndependentTimeConductor from '@/plugins/timeConductor/independent/IndependentTimeConductor.vue';
 import tooltipHelpers from '../../api/tooltips/tooltipMixins';
 
-const SupportedViewTypes = ['plot-stacked', 'plot-overlay', 'bar-graph.view', 'time-strip.view'];
+const SupportedViewTypes = [
```
Comment (jvigliotta) [inline src/ui/layout/BrowseBar.vue:145]: Do we need to add this to ObjectFrame.vue too? I think that is when it's in a display layout...

**Review by jvigliotta — CHANGES_REQUESTED** [review]
Couple questions and changes! Nice work!

File: src/ui/layout/BrowseBar.vue (hunk 4 above)
Comment (scottbell) [reply src/ui/layout/BrowseBar.vue:145]: Yes!

File: src/plugins/imagery/components/ImageryView.vue [hunk 5]
```diff
@@ -375,14 +375,11 @@ export default {
       return age < cutoff && !this.refreshCSS;
     },
     canTrackDuration() {
-      let hasClock;
       if (this.timeContext) {
-        hasClock = this.timeContext.clock();
+        return this.timeContext.isRealTime();
```
Comment (akhenry) [inline src/plugins/imagery/components/ImageryView.vue:379]: We should default this.timeContext to the global time context (ie. openmct.time) and avoid this boilerplate. Indepentent time contexts and the global time context have the same interface for this reason.

**Review by akhenry — APPROVED** [review]

**Pull request merged**